### ISSUE TITLE
About Anomaly Segmentation pictures

### ISSUE THREAD

**Issue by JackyLin7414** [DarioFontanel/PAnS#6]
> Hi Jacky, Qualitative results are not implemented in this code base. We created them offline, by loading the trained weights of the model and visualizing the confidence. We used [imshow](https://matplotlib.org/3.5.0/api/_as_gen/matplotlib.pyplot.imshow.html) as follows:
> 
> `imshow(confidence, interpolation='bilinear', cmap=matplotlib.cm.Blues, vmin=-1, vmax=1) `
> 
> Dario

Hi, @DarioFontanel 

Thank you for the quick reply.

Could you tell me where to put this line?
I can't find the confidence from this code. Where can I find it?

Is the `confidence `from this line array-like or PIL image?
`imshow(confidence, interpolation='bilinear', cmap=matplotlib.cm.Blues, vmin=-1, vmax=1)`

Thank you
Jacky

_Originally posted by @JackyLin7414 in https://github.com/DarioFontanel/PAnS/issues/5#issuecomment-1080255522_

**Comment by JackyLin7414** [comment]
Hi, @DarioFontanel

I am running this code from cmd not jupyter-notebook so I cannot use below command to get the picture.
`imshow(confidence, interpolation='bilinear', cmap=matplotlib.cm.Blues, vmin=-1, vmax=1)`

Can you tell me how can I do?

Thank you. I appreciate it.
Jacky

**Comment by DarioFontanel** [comment]
Hi @JackyLin7414,
Sorry for the late reply, I missed the notification. For confidence I mean basically the logits produced by the network. It is not something that I used during training or testing the model, it has been used just for creating the qualitative results. If you cannot use jupyter-notebook, you could try to use matplotlib and save the resulting .png images.

Dario

**Comment by JackyLin7414** [comment]
Thank you very much. I appreciate it.
